src/Auth0.php

```diff
@@ -245,6 +253,11 @@ public function __construct(array $config)
             throw new CoreException('Invalid domain');
         }
 
+        $this->discoveryDocumentUri = $config['discovery_document_uri'] ?? $_ENV['AUTH0_DISCOVERY_DOCUMENT_URI'] ?? null;
+        if (empty($this->discoveryDocumentUri)) {
+            throw new CoreException('Invalid discovery document');
```

Comment:
This is a breaking change and we're not planning on releasing another major anytime soon. Will need to fallback on the existing URLs if, for some reason, we can't do discovery.